lib/node_modules/@stdlib/math/base/special/roundnf/lib/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Round a single-precision floating-point number to the nearest multiple of `10^n`.
+*
+* @module @stdlib/math/base/special/roundnf
+*
+* @example
+* var roundnf = require( '@stdlib/math/base/special/roundnf' );
+*
+* // Round a value to 2 decimal places:
+* var v = roundnf( 3.141592, -2 );
+* // returns 3.140000104904175
+*
+* @example
+* var roundnf = require( '@stdlib/math/base/special/roundnf' );
+*
+* // If n = 0, `roundnf` behaves like `roundf`:
+* var v = roundnf( 3.141592, 0 );
```

Comment:
```suggestion
* var v = roundnf( 3.1415927410125732, 0 );
```

Keeping the same example as above here too, for consistency.